docs/data/routes/docs/techniques/redirection/headless-ssr/en.md

```diff
@@ -0,0 +1,134 @@
+2. [Use the redirect template for new and existing pages.](#use-the-redirect-template-with-new-and-existing-pages)
+3. [Customize the `node-headless-ssr-proxy` sample to handle redirects.](#customize-the-node-headless-ssr-proxy-sample-to-handle-redirects)
+
+## Create a page template with a redirect field
+
+In your Sitecore instance:
+
+1. [Create a data template]((https://doc.sitecore.com/developers/101/sitecore-experience-manager/en/create-a-data-template.html)) called **Redirect Route** that inherits from the `App Route` template.
+2. Under the **Builder** tab, [add a field to the data template](https://doc.sitecore.com/developers/101/sitecore-experience-manager/en/add-a-field-to-a-data-template.html) of type **General Link** field named **Destination** and **Save**. 
+
+## Use the redirect template with new and existing items
+
+1. To create new items with redirect capability, [insert new items in the application's content folder using the new Redirect Route template](https://doc.sitecore.com/users/101/sitecore-experience-platform/en/create-an-item-in-the-content-editor.html#UUID-33faf7af-c5e7-5bc5-5ad9-42325eeb16d2_id__Insert_an_item_1). 
+2. To apply the template to existing items, [edit or change the template that an item is based on](https://doc.sitecore.com/users/101/sitecore-experience-platform/en/edit-or-change-the-template-that-an-item-is-based-on.html).
+
+## Customize the `node-headless-ssr-proxy` sample to handle redirects
+
+> This walkthrough assumes you have followed the ["Headless SSR via `sitecore-jss-proxy`" documentation](/docs/techniques/ssr/headless-mode-ssr) to set up the headless server-side rendering of your application.
+
+1. In the  `/.env` file, define environment variables: 
+
+   ```
+   SITECORE_JSS_APP_NAME="JssReactWeb"
+   SITECORE_API_HOST="https://cm.jss16.localhost"
+   SITECORE_HOSTNAME="https://react.jss16.localhost"
+   SITECORE_API_KEY="{5379B475-A885-4B62-9A51-EE79E5434650}"
+   SITECORE_GRAPHQL_ENDPOINT="https://cm.jss16.localhost/api/jssreactweb?sc_apikey={5379B475-A885-4B62-9A51-EE79E5434650}"
+   DEFAULT_LANGUAGE="en"
+   ```
+
+2. Install the `dotenv` npm package to load the environment variables into the Node `process.env`:
+
+   ```
+   npm install --save-dev dotenv
+   ```
+
+3. In `/config.js`:
+
+   1. Require and configure `dotenv`:
+
+   ```javascript
+   require('dotenv').config();
+   ```
+
+   2. Declare the following constants: 
+
+      ```javascript
+      const graphQLEndpoint = process.env.SITECORE_GRAPHQL_ENDPOINT || serverBundle.graphQLEndpoint || `${apiHost}/api/${appName}`;
+      
+      const defaultLanguage = process.env.DEFAULT_LANGUAGE || serverBundle.defaultLanguage || 'en';
+      
+      const hostname = process.env.SITECORE_HOSTNAME || serverBundle.hostname || apiHost;
+      ```
+
+   3. For the `config` module, define a new member function `getRedirects`. This example uses a GraphQL query: 
```

Comment:
Link to config module: https://github.com/Sitecore/jss/blob/master/samples/node-headless-ssr-proxy/config.js

Reply:
At this step, we assume they already have a local copy of the file (see note at line 23). Because of that, I wouldn't link to a web page, just tell them which file they are working on inside the directory they created while setting up headless server-side rendering.